### PR TITLE
36969: add operator dependencies

### PR DESCRIPTION
**Pull request by cheewaio**

### CODE REVIEW

File: deploy/olm-catalog/ibm-catalog-ui-operator/3.6.0/ibm-catalog-ui-operator.v3.6.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -68,6 +68,37 @@ metadata:
...
+                  },
+                  {
+                    "name":"ibm-platform-api-operator"
+                  },
+                  {
+                    "name":"ibm-helm-api-operator"
+                  },
+                  {
+                    "name":"ibm-helm-repo-operator"
+                  }
```
Comment (bitscuit) [inline deploy/olm-catalog/ibm-catalog-ui-operator/3.6.0/ibm-catalog-ui-operator.v3.6.0.clusterserviceversion.yaml:96]: are cert manager and mongodb not dependencies?

**Review by bitscuit — COMMENTED** [review]
comments above

File: deploy/olm-catalog/ibm-catalog-ui-operator/3.6.0/ibm-catalog-ui-operator.v3.6.0.clusterserviceversion.yaml (hunk 1 above)
Comment (cheewaio) [reply deploy/olm-catalog/ibm-catalog-ui-operator/3.6.0/ibm-catalog-ui-operator.v3.6.0.clusterserviceversion.yaml:96]: we only need to list direct dependencies

**Pull request merged**